uniforms.md

```diff
@@ -80,6 +80,23 @@ uniform int heldBlockLightValue2;
 * ❌ ShadersMod
 * ✔️ OptiFine
 
+### Player Mood
+
+Player Mood is a float that is in the range of (0.0-1.0), it increases the longer a player stays underground.
+It's value in percentage can be viewed in the debug screen(F3).
+https://minecraft.gamepedia.com/Ambience#Cave_ambience
```

Comment:
I would like to avoid copying content from OptiFine's documentation if possible. This appears to be a copy of [this line](https://github.com/sp614x/optifine/blob/master/OptiFineDoc/doc/shaders.txt#L183) in the OF docs.
```suggestion
The [Minecraft Wiki](https://minecraft.gamepedia.com/Ambience#Cave_ambience) has documented this value:

> Cave ambience occurs based on a "mood" percent value between 0–100. The mood increases when the player is underground and/or in dark place, and decreases otherwise. When the mood reaches 100%, one of the sounds plays randomly, resetting the mood to 0% and thus restarting the cycle. The deeper the player is and the darker it is, the quicker the mood increases. Mood and its current value can be seen in the debug screen.

In this case, 100% corresponds to a float value of 1.0, 0% corresponds to a value of 0.0, and so on.
```